editors/code/package.json

```diff
@@ -362,6 +357,16 @@
                     },
                     "default": [],
                     "description": "List of features to activate"
+                },
+                "rust-analyzer.cargoFeatures.loadOutDirsFromCheck": {
+                    "type": "boolean",
+                    "default": false,
+                    "markdownDescription": "Run `cargo check` on startup to get the correct value for package OUT_DIRs"
+                },
+                "rust-analyzer.cargoFeatures.outDirOverrides": {
```

Comment:
Do we really need `outDirOverrides` if we have `loadOutDirsFromCheck`?

Reply:
I can remove it, it's probably not necessary, and fairly simple to add again if somebody _really_ needs it badly.